drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1779,7 +1779,63 @@ void TextureStorage::_create_render_target_backbuffer(RenderTarget *rt) {
 		glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
 	}
 }
+void GLES3::TextureStorage::copy_scene_to_backbuffer(RenderTarget *rt, const bool uses_screen_texture, const bool uses_depth_texture) {
+	if (rt->backbuffer != 0 && rt->backbuffer_depth != 0) {
+		return;
+	}
+
+	Config *config = Config::get_singleton();
+	bool use_multiview = rt->view_count > 1 && config->multiview_supported;
+	GLenum texture_target = use_multiview ? GL_TEXTURE_2D_ARRAY : GL_TEXTURE_2D;
+
+	glGenFramebuffers(1, &rt->backbuffer_fbo);
```

Comment:
```suggestion
	if (rt->backbuffer_fbo == 0) {
		glGenFramebuffers(1, &rt->backbuffer_fbo);
	}
```
We need to check if the framebuffer already exists. If it does, we do not want to generate a new one and overwrite the old one.